ViewModels/PanZoomViewModel.cs

```diff
@@ -85,7 +85,7 @@ public void StartCreatingRegion(Point p)
                     p.Y > r.ScaledY && p.Y < (r.Y + r.ScaledHeight))
                 {
                     _currently_creating_region = null;
-                    return;
+                    //return; fixes the problem that no regions can be created if one exists at 0x0
```

Comment:
In general, please remove rather than commenting out.

The idea here was that we want to avoid creating a region if the point clicked is within the bounds of an already existing region.  Can you explain what you mean by a region that exists at 0x0?

I think not having this will allow for overlapping regions which I was trying to avoid.

Reply:
if there is a regin in the upper left corner (X=0, Y=0). It is not possible to create more regions, which is more than annoying.
i will check this again, but without the `return` there seems to be no problems.

Reply:
ok the bug is that it uses the scaled versions, but then it can also not be triggered because you click on the region above.
Should I fix the error or delete the code?

Did you mean overlapping in general?
then it should probably be in the update logic, but overlapping is also useful when a game uses the same texture for multiple controllers.

Reply:
> ok the bug is that it uses the scaled versions, but then it can also not be triggered because you click on the region above.
Should I fix the error or delete the code?

Let me get back to you.  I'll look at it more tomorrow

> Did you mean overlapping in general?
then it should probably be in the update logic, but overlapping is also useful when a game uses the same texture for multiple controllers.

Yes, I meant overlapping in general.  I can see some edge cases where overlapping may be useful but right now the tool has no way to show multiple regions in one place (or any way to pick the desired region).  If we allow overlapping regions now, I think it may make the tool harder to use (which is why I opted for this approach).

Reply:
unfortunately, the code does not prevent it, it only prevents new regions from being created in another region. which is currently not possible. Currently I have no plans to prevent overlapping in general, maybe at a later time.